blog/01-30-23-Build-Your-Own-Full-Stack-NFT-Marketplace-on-Celo/Build-Your-Own-Full-Stack-NFT-Marketplace-on-Celo.md

```diff
@@ -0,0 +1,1263 @@
+---
+title: Build Your Own Full-Stack NFT Marketplace on Celo
+description: Build a full stack Nft Marketplace on the celo block-chain using ipfs and web3.storage for metadata storage.
+authors:
+  - name: ✍️ Jonathan Iheme
+url: https://github.com/4undRaiser
+image_url: https://avatars.githubusercontent.com/u/87926451?s=96&v=4
+tags: [celosage, nft, solidity, celo]
+hide_table_of_contents: false
+slug: "/tutorials/Build-Your-Own-Full-Stack-NFT-Marketplace-on-Celo"
+---
+
+
+
+# Build Your Own Full-Stack NFT Marketplace on Celo
```

Comment:
Please remove the title from here. There is no need to add title. It will be added to the side automatically from the frontmatter.